native/src/components/NewsHeader.tsx

```diff
@@ -36,6 +36,11 @@ const HeaderContainer = styled.View`
   margin-bottom: 12px;
 `
 
+const StyledIcon = styled(Icon)`
+  width: 110px;
+  height: 34px;
+`
```

Comment:
Maybe a comment to explain the magic numbers?

Reply:
I don't like the increase of the sizes as I think the tabs are then in relation to the caption too big. Feel free to open a new Issue though. Also, I just took the sizes from before, nothing new here, so I also don't know where the magic numbers are coming from.